src/developer/web-api/route.md

```diff
@@ -18,70 +18,105 @@ X-Forwarded-User: <username>
 
 For the examples here we will be using the the [Echo API](https://learning.postman.com/docs/developer/echo-api/) from Postman which just returns what you send to it (including body in the case of `POST`).
 
-### GET Route without authentication
+### Running a route
+
+The following is an example of a JSON request creating a route:
 
 ```json
 {
-  "name": "postman-get",
-  "code": "postman-get",
+  "name": "postman",
+  "code": "postman",
   "disabled": false,
-  "url": "https://postman-echo.com/get"
+  "url": "https://postman-echo.com/"
```

Comment:
```suggestion
  "url": "https://postman-echo.com/get"
```

We should keep this pointing at postman-echo/get - the get endpoint echoes the request sent with a GET method, which is good for testing a route.  Using `postman-echo.com` without a path returns the HTML of the `postman-echo` website